fastlane/Fastfile

```diff
@@ -418,19 +418,12 @@ platform :ios do
     en_lproj = 'WordPress/Resources/en.lproj'
     ios_generate_strings_file_from_code(
       paths: ['WordPress/', 'Pods/WordPress*/', 'Pods/WPMediaPicker/', 'WordPressShared/WordPressShared/', 'Pods/Gutenberg/'],
-      exclude: ['*Vendor*', 'WordPress/WordPressTest/I18n.swift', 'WordPress/WordPressStatsWidgets/Views/Localization/AppLocalizedString.swift'],
+      exclude: ['*Vendor*', 'WordPress/WordPressTest/I18n.swift', 'WordPress/WordPressStatsWidgets/Views/Localization/AppLocalizedString.swift', 'WordPress/Classes/Utility/AppLocalizedString.swift'],
```

Comment:
@AliSoftware – I needed to make this change to get this method to work – LMK if that looks correct? I think we could potentially also use a wildcard here, but I kept it simple to be safe.

Reply:
The file was actually moved from its previous location in `WordPress/WordPressStatsWidgets/Views/Localization/AppLocalizedString.swift` to the new one you added, so you should just replace the old one actually:
```suggestion
      exclude: ['*Vendor*', 'WordPress/WordPressTest/I18n.swift', 'WordPress/Classes/Utility/AppLocalizedString.swift'],
```

(I should have done that in https://github.com/wordpress-mobile/WordPress-iOS/pull/17630 alongside the file move but totally missed it back then)